python/raydp/spark/ray_cluster.py

```diff
@@ -131,11 +131,22 @@ def _prepare_spark_configs(self):
         self._configs[SPARK_RAY_LOG4J_FACTORY_CLASS_KEY] = versions.RAY_LOG4J_VERSION
 
         if SPARK_LOG4J_CONFIG_FILE_NAME not in self._configs:
+            # If the config is not passed in by the user, check
+            # by environment variable SPARK_LOG4J_CONFIG_FILE_NAME. This
+            # will give the system admin/infra team a chance to set the
+            # value for all users.
             self._configs[SPARK_LOG4J_CONFIG_FILE_NAME] =\
-                versions.SPARK_LOG4J_CONFIG_FILE_NAME_DEFAULT
+                os.environ.get("SPARK_LOG4J_CONFIG_FILE_NAME",
```

Comment:
Can you remove double quotes from "SPARK_LOG4J_CONFIG_FILE_NAME" ? SPARK_LOG4J_CONFIG_FILE_NAME evaluates to "spark.log4j.config.file.name" which is used in configs in init_spark function. It'll make config name consistent.

Reply:
What I am trying to do here is if there is an environment variable `SPARK_LOG4J_CONFIG_FILE_NAME` defined, set the value of `spark.log4j.config.file.name` to the value defined by that env var. So I think the logic here is correct.

Reply:
Yes, I know.  
So you want to use different names for the same thing? SPARK_LOG4J_CONFIG_FILE_NAME for env and "spark.log4j.config.file.name" for configs in init_spark. And they both are for the same log4j file name.
Is it better to use same name, like "spark.log4j.config.file.name"?

Reply:
I don't think we can use dot(.) in environment variable name, at least [not for bash](https://unix.stackexchange.com/questions/93532/exporting-a-variable-with-dot-in-it). Also using lowercase env name in general is not a convention.

I updated the readme, you can see the example usage.

Reply:
ok, I see.